setup.py

```diff
@@ -45,10 +48,10 @@
     tests_require=TEST_REQUIREMENTS,
     test_suite="tests.settings.run",
     dependency_links=[
-        "http://github.com/divio/django-cms/tarball/release/4.0.x#egg=django-cms-4.0.0",
-        "http://github.com/divio/djangocms-versioning/tarball/master#egg=djangocms-versioning-0.0.23",
-        "http://github.com/FidelityInternational/djangocms-version-locking/tarball/master#egg=djangocms-version-locking-0.0.13", # noqa
-        "https://github.com/divio/djangocms-text-ckeditor/tarball/support/4.0.x#egg=djangocms-text-ckeditor-4.0.x",
-        "https://github.com/divio/aldryn-forms/tarball/master#egg=aldryn-forms"
+        "https://github.com/divio/aldryn-forms/tarball/master#egg=aldryn-forms-5.0.1",
+        "https://github.com/divio/django-cms/tarball/release/4.0.x#egg=django-cms-4.0.0",
+        "https://github.com/divio/djangocms-versioning/tarball/master#egg=djangocms-versioning-0.0.23",
+        "https://github.com/jonathan-s/djangocms-version-locking/tarball/setup-django22#egg=djangocms-version-locking-0.0.13", # noqa
```

Comment:
Needs to be changed back, once version-locking is ready. 
```suggestion
        "http://github.com/FidelityInternational/djangocms-version-locking/tarball/master#egg=djangocms-version-locking-0.0.13", # noqa
```